guides/user/kubernetes-v2/best-practices/index.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: single
+title:  "Best Practices for the Kubernetes Provider V2"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+The Kubernetes Provider V2 enables a wide variety of ways to deploy your
+manifests into Kubernetes clusters. This page provides best-practices for doing
+so.
+
+## Deploy Docker Images by Digest
+
+If your registry exposes image changes by
+[digest](https://docs.docker.com/registry/spec/api/#content-digests), we
+recommend deploying images by their digest rather than tag, e.g.
+`gcr.io/my-image@sha256:95ff090...` rather than `gcr.io/my-image:v1`.
+
+The reasoning is: the digest is a content-addressable way of referencing
+your image, since it's derived from the hash of the image contents. If your
+manifest points to an image by its digest, you know that each time it's
+deployed, it will point to the same contents (binary, dependencies, config,
+etc...). If you rely on the tag, it's entirely possible that deploying the same
+Manifest twice can have different results.
+
+There are two ways to achieve this in Spinnaker:
+
+1. Template your manifests outside of Spinnaker to point to the digest you want
```

Comment:
Maybe elaborate, for some set of less K8s-savvy users, what it means to template one's manifest. Or just link to something if there's anything good to link to.